mage_ai/server/app.py

```diff
@@ -302,10 +303,13 @@ def update_pipeline(id):
     feature_set_id = pipeline.metadata.get('feature_set_id')
     if feature_set_id is not None:
         feature_set = FeatureSet(id=feature_set_id)
-        df_transformed = clean_pipeline.transform(feature_set.data_orig, auto=False)
+        df = feature_set.data_orig
+        ctypes = feature_set.metadata.get('column_types', {})
+        df = clean_dataframe(df, ctypes, False)
```

Comment:
@skunichetty This throws error when deleting a "remove_column" action. The column_types doesn't include the deleted column name.

Reply:
Cool. Let's push the fix first.